costar_dataset/block_stacking_reader_torch.py

```diff
@@ -1016,6 +1199,25 @@ def ConvertImageListToNumpy(data, format='numpy', data_format='NHWC'):
                     # print("y = ", y)
             except IOError as ex:
                 print('Error: Skipping file due to IO error when opening ' + example_filename + ': ' + str(ex))
+            # If features to extract is image_m_image_n then return batch (two images and label which is the interval)           
+            if (self.data_features_to_extract is not None and 'image_m_image_n' in self.data_features_to_extract):
+                if(self.visual_mode):
+                    return (np.array(rgb_images_resized), None)
+                batch = generate_temporal_distance_training_data(np.array(rgb_images_resized))
+                return batch
+
+            if (self.data_features_to_extract is not None and 'image_n_vec_xyz_aaxyz_nsc_q_dq_gripper' in self.data_features_to_extract): 
+                if(self.visual_mode):
+                    num_joints_concatenated = 10        #TODO Do we want to include this as one of the private variables and then set it during the creation of the class object? 
```

Comment:
I don't think this todo is resolved.

Reply:
I wanted to ask you about this. How do you think we should set this variable?